### PR TITLE
Can save recursive data.

### PR DESCRIPTION
**Pull request by ghost**
This can save recursive data.

### CODE REVIEW

File: js/rpg_core/JsonEx.js [hunk 1]
```diff
@@ -87,12 +154,16 @@ JsonEx._encode = function(value, depth) {
  * @static
  * @method _decode
  * @param {Object} value
+ * @param {Object} circular
```
Comment (krmbn0576) [inline js/rpg_core/JsonEx.js]: {Array} ?

Comment (ghost) [reply js/rpg_core/JsonEx.js]: Yes. modified.

**Comment by krmbn0576** [comment]
Review ended. LGTM
(I think `@m` doesn't seem necessary, but you don't have to worry about such details.)

**Comment by ghost** [comment]
What do you mean? `@m` needs for traversing mark.

**Comment by krmbn0576** [comment]
Traversing mark is enough because `@c` is non-zero when it is marked.
But this is hard to understand, so you don't have to worry about it.
Sorry, from now on I will not write about poor comments. 😢

**Comment by ghost** [comment]
I see. I consider it (This is why `@c` is not zero-origin). and first Implementation is different. It can think as dead code. but `@m` intention is clear, so I prefer leave this.

**Pull request merged**